### DISCUSSION TITLE
crash osu! lazer when switching another tabs on Android

### DISCUSSION THREAD

**Discussion by Halozhan** [ppy/osu#17151] [Q&A] [answered]
I have tested osu! on Samsung Galaxy S21 Ultra Android 12.
The default option single-threaded mode seems stable and no freeze when i switching other tabs or go home menu. But on multi-threaded mode, it ring the alert and freeze osu then crash. Similar as #6295 [6295](https://github.com/ppy/osu/issues/6295) 
But I guess this crash from multi thread side.

**Comment by bdach** [comment, marked as answer]
duplicate of https://github.com/ppy/osu-framework/issues/4994
